test/components/developers/unseeded_warning_component_test.rb

```diff
@@ -0,0 +1,21 @@
+require "test_helper"
+
+module Developers
+  class UnseededWarningComponentTest < ViewComponent::TestCase
+    test "hidden when unseedable" do
+      render_inline UnseededWarningComponent.new(seedable: false)
+      assert_no_text "Run bin/rails db:seed"
+    end
+
+    test "hidden when seedable but developers exist" do
+      render_inline UnseededWarningComponent.new(seedable: true)
+      assert_no_text("Run bin/rails db:seed")
```

Comment:
Let's consolidate using parenthesis or not for these assertions.

Reply:
Nice catch. I removed them to be consistent 👍